controllers/operatorconfig/operatorconfig_controller.go

```diff
@@ -19,11 +19,22 @@ package operatorconfig
 import (
 	"context"
 
+	"github.com/barkimedes/go-deepcopy"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
 	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	operatorv1alpha1 "github.com/IBM/operand-deployment-lifecycle-manager/api/v1alpha1"
 	deploy "github.com/IBM/operand-deployment-lifecycle-manager/controllers/operator"
+	olmv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
```

Comment:
We might need to move this line to the upper section

```golang
	"sigs.k8s.io/controller-runtime/pkg/source"
        olmv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"

	operatorv1alpha1 "github.com/IBM/operand-deployment-lifecycle-manager/api/v1alpha1"
	deploy "github.com/IBM/operand-deployment-lifecycle-manager/controllers/operator"
```

Reply:
yea, just found it. Fixed